roles/edpm_kernel/tasks/kernelargs.yml

```diff
@@ -100,13 +100,23 @@
         - /boot/efi/EFI/fedora
 
     - name: Generate grub config
-      ansible.builtin.command: "grub2-mkconfig -o /boot/grub2/grub.cfg"
+      ansible.builtin.command: >-
+        grub2-mkconfig -o /boot/grub2/grub.cfg
+        {{ '--update-bls-cmdline'
+        if ansible_facts['distribution'] == 'RedHat'
+        and ansible_facts['distribution_version'] | float >= 9.3
+        else '' }}
       register: _grub2_mkconfig
       changed_when: _grub2_mkconfig.rc == 0
       failed_when: _grub2_mkconfig.rc != 0
 
     - name: Generate EFI grub config
-      ansible.builtin.command: "grub2-mkconfig -o {{ item.stat.path }}/grub.cfg"
+      ansible.builtin.command: >-
+        grub2-mkconfig -o /boot/grub2/grub.cfg
```

Comment:
@eshulman2  I think grup2-mkconfig command '-o' option should be same grub2-mkconfig -o {{ item.stat.path }}/grub.cfg and only new arg is the additional.